lib/local-storage.js

```diff
@@ -21,6 +21,20 @@ function Storage(config) {
   var self = Object.create(Storage.prototype)
   self.config = config
   self.logger = Logger.logger.child({ sub: 'fs' })
+  //TODO -- Figure out if there is some concept of default config?
+  // If a store plugin hasn't been configured, put in the local-fs one.
+  if(self.config.store == undefined) {
+    self.config.store = {'./disk-store': 'disk'}
+  }
+  var store = load_plugins(config, self.config.store, {
+    config : config,
+    logger : self.logger,
+  }, function(p) {
+    return true
+  })
+  // TODO -- support multiple stores? I vote no. Bomb out in this case
```

Comment:
Private packages stored in the database with replication, public packages stored on the disk nobody cares about. This is an example of how people would like to use 2 storages at the same time.